package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@blackbaud/skyux-cli",
+  "name": "@blackbaud/skyux-sdk-cli",
```

Comment:
Was this the name name we wanted or `@skyux-sdk/cli`?

Reply:
Good catch!